csatar-plugins/csatar/updates/TestData.php

```diff
@@ -3,17 +3,53 @@
 use Seeder;
 use Csatar\Csatar\Models\Association;
 use Csatar\Csatar\Models\District;
+use Csatar\Csatar\Models\Patrol;
 use Csatar\Csatar\Models\Team;
+use Csatar\Csatar\Models\Troop;
 
 class TestData extends Seeder
 {
+    public const DATA = [
+        'district' => [
+            'Nógrád',
+            'Csík',
+            'Észak-Erdély',
+            'Háromszék',
+        ],
+        'team' => [
+            'Nógrádi első próba csapat',
+            'Szent István',
+            'Zöld Péter',
+            'Élthes Alajos',
+            'Hollósy Simon',
+            'Szent György',
+            'Nagyboldogasszony',
+        ],
+        'troop' => [
+            'Madarak',
+            'Virágok',
+            'Madarak',
+            'Virágok',
```

Comment:
Ez szándékosan vagy kétszer?

Reply:
Igen, mert különböző csapatokhoz tartóznak. Gondoltam, hogy legyen ilyen teszt adat is.